src/RfbSocket.cpp

```diff
@@ -7,7 +7,12 @@
 #include <qtcpsocket.h>
 #include <qrect.h>
 
+#ifdef WIN32
+#include "WinSock2.h"
+#pragma comment(lib, "Ws2_32.lib")
+#else
 #include <arpa/inet.h>
+#endif
```

Comment:
This project might be usable on any platform, where Qt is sitting on OpenGL - maybe it is even possible to replace the qt_gl_read_framebuffer call by some RHI API.

However I narrowed the project for the EGLFS ( = Unixoid operating systems ) platform - simply because I assumed, that on other platforms you find satisfying VNC solutions available on the operating level.

Did I miss something ?

Reply:
No, I don't think you did, it's just that the project I'm working on can be built for windows. I was curious if I could get your project to build on windows too. It simplyfies testing a lot if one doesn't have to constantly deploy everything to an embedded device.